lib/Dancer/Plugin/SimpleCRUD.pm

```diff
@@ -1132,20 +1147,34 @@ SEARCHFORM
         my $opposite_order_by_direction
             = ($order_by_direction eq "asc") ? "desc" : "asc";
 
+        # Get a list of all columns (normal, and custom_columns), then assemble
+        # the names and labels to pass to HTML::Table::FromDatabase
+        my @all_cols = map { $_->{COLUMN_NAME} } @$columns;
+        push @all_cols, keys %{ $args->{custom_columns} }
+            if exists $args->{custom_columns};
         %columns_sort_options = map {
-            my $col_name       = $_->{COLUMN_NAME};
-            my $col = $args->{labels}{$col_name} || $col_name;
+            my $col_name       = $_;
             my $direction      = $order_by_direction;
             my $direction_char = "";
+            my $friendly_name  = $col_name;
+            if ($args->{labels}{$col_name}) {
+                $friendly_name = $args->{labels}{$col_name};
+            } else {
+                for ($friendly_name) {
+                    lc($friendly_name);
```

Comment:
I think there's a missing assignment here. I'm getting warnings from make test:
t/00-load.t ....... Useless use of lc in void context at .../Dancer-Plugin-SimpleCRUD/blib/lib/Dancer/Plugin/SimpleCRUD.pm line 1164.